hicommonwealth/commonwealth
# User-Onboarding: Added initial CreateAccount Modal

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/6641

## Description of Changes
- Adds initial step of CreateAccount Modal

## "How We Fixed It"
N/A

## Test Plan
This PR only adds new components, so not much to test.

- Enable `FLAG_USER_ONBOARDING_ENABLED` in `.env`.
- Add `type='create-account'` to `<AuthModal/>` instance in `SubLayoutHeader.tsx` and open the modal, verify it matches [figma](https://www.figma.com/file/I5jbkBPzNrbYMHYQJb6CNY/Onboarding?type=design&node-id=802-5224&mode=design&t=zakjlow5RcFRlS0c-4) or apply this diff
```diff
diff --git a/packages/commonwealth/client/scripts/views/components/SublayoutHeader/SublayoutHeader.tsx b/packages/commonwealth/client/scripts/views/components/SublayoutHeader/SublayoutHeader.tsx
index 4564d26340..c58647b700 100644
--- a/packages/commonwealth/client/scripts/views/components/SublayoutHeader/SublayoutHeader.tsx
+++ b/packages/commonwealth/client/scripts/views/components/SublayoutHeader/SublayoutHeader.tsx
@@ -59,6 +59,7 @@ export const SublayoutHeader = ({
         open={isFeedbackModalOpen}
       />
       <AuthModal
+        type='create-account'
         onClose={() => setIsAuthModalOpen(false)}
         isOpen={isAuthModalOpen}
       />

```

## Deployment Plan
N/A

## Other Considerations
N/A
